### DISCUSSION TITLE
Possible to configure SELL behaviour to act upon FIFO unit-price, not Average unit-price?

### DISCUSSION THREAD

**Discussion by 6ameDev** [ghostfolio/ghostfolio#1711] [Q&A] [answered]
I tried searching for discussions/issues regarding the same, but couldn't find anything relevant.

So I'm trying to understand if there is a way to configure Ghostfolio to evaluate the SELL transactions by deducting the unit-prices in the FIFO order instead of using avg unit-price to reduce the total invested amount (in order to derive the NEW avg unit-price).

NOTE: This does not affect the `Value` of the holdings in any way, its only a difference in how the `Performance` is perceived.

**Example:**

| SYMBOL | QUANTITY | UNIT-PRICE | TYPE | DATE |
|---|---|---|---|---|
| XYZ | 10 | 100 | BUY | 01-02-2022 |
| XYZ | 10 | 200 | BUY | 01-03-2022 |
| XYZ | 10 | 300 | BUY | 01-04-2022 |
| XYZ | 9 | 400 | SELL | 01-05-2022 |

**Before 01-05-2022**
Total Invested Amount = 6000
Total Quantity = 30
Avg Unit-Price = 200

**After 01-05-2022**

**Approach: Avg Unit-Price**
Total Invested Amount = 6000 - (9*200) = 4200
Total Quantity = 30 - 9 = 21
Avg Unit-Price = 4200 / 21 = **200**

**Approach: FIFO Unit-Price**
Total Invested Amount = 6000 - (9*100) = 5100
Total Quantity = 30 - 9 = 21
Avg Unit-Price = 5100 / 21 = **242.85**

**Comment by dtslvr** [comment, marked as answer]
> So I'm trying to understand if there is a way to configure Ghostfolio to evaluate the SELL transactions by deducting the unit-prices in the FIFO order instead of using avg unit-price to reduce the total invested amount

No, at the moment it is not possible to configure this.
